receiver/azureblobrehydrationreceiver/receiver.go

```diff
@@ -90,9 +101,13 @@ func newTracesReceiver(id component.ID, logger *zap.Logger, cfg *Config, nextCon
 	return r, nil
 }
 
+// factor of buffered channel size
+// number of blobs to process at a time is blobChanSize * batchSize
+const blobChanSize = 5
```

Comment:
Cool, this seems like a good size, its performing well for me. In the future maybe we can expose it. I don't see a need to right now.